### PR TITLE
BerryIO: Release app as uncertified on the portal 

### PR DESCRIPTION
**Pull request by mishalp-crest**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
- [ ] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: release_notes.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<b>BerryIO Release Notes - Published by Splunk February 07, 2018</b>
+<br><br>
+<b>Version 1.0.8 - Released February 07, 2018</b>
+<ul>
+<li>App action views and Logo updates</li>
```
Comment (alexa-phantom) [inline release_notes.html]: We already released this release note, but `Logo` can be `logo

File: berryio_connector.py [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+    # The first param is the input json file
+    with open(sys.argv[1]) as f:
+
+        # Load the input json file
+        in_json = f.read()
+        in_json = json.loads(in_json)
+        print(json.dumps(in_json, indent=' ' * 4))
+
+        # Create the connector class object
+        connector = BerryIOConnector()
```
Comment (alexa-phantom) [inline berryio_connector.py:391]: In the future if we plan to support this ourselves, we'd want to add our usual debug lines

**Review by alexa-phantom — DISMISSED** [review]

**Review by ricardo-phantom — DISMISSED** [review]

File: berryio_connector.py (hunk 2 above)
Comment (mishalp-crest) [reply berryio_connector.py:391]: done

File: release_notes.html (hunk 1 above)
Comment (mishalp-crest) [reply release_notes.html]: done

File: berryio_connector.py [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+    argparser.add_argument('-p', '--password', help='password', required=False)
+
+    args = argparser.parse_args()
+    session_id = None
+
+    if args.username and args.password:
+        login_url = BaseConnector._get_phantom_base_url() + "login"
+        try:
+            print("Accessing the Login page")
+            r = requests.get(login_url, verify=False)
```
[the remark this answers is not in the document]
Comment (mishalp-crest) [reply berryio_connector.py]: done

File: berryio_connector.py [hunk 4]
```diff
@@ -0,0 +1,401 @@
...
+            csrftoken = r.cookies['csrftoken']
+            data = {'username': args.username, 'password': args.password, 'csrfmiddlewaretoken': csrftoken}
+            headers = {'Cookie': 'csrftoken={0}'.format(csrftoken), 'Referer': login_url}
+
+            print("Logging into Platform to get the session id")
+            r2 = requests.post(login_url, verify=False, data=data, headers=headers)
```
[the remark this answers is not in the document]
Comment (mishalp-crest) [reply berryio_connector.py]: done

File: berryio_connector.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (mishalp-crest) [reply berryio_connector.py]: done

File: berryio_connector.py [hunk 5]
```diff
@@ -0,0 +1,401 @@
...
+    argparser.add_argument('-p', '--password', help='password', required=False)
+
+    args = argparser.parse_args()
+    session_id = None
+
+    if args.username and args.password:
+        login_url = BaseConnector._get_phantom_base_url() + "login"
+        try:
+            print("Accessing the Login page")
+            r = requests.get(login_url, verify=False, timeout=60)
```
[the remark this answers is not in the document]
Comment (mishalp-crest) [reply berryio_connector.py]: done

File: berryio_connector.py [hunk 6]
```diff
@@ -0,0 +1,401 @@
...
+            csrftoken = r.cookies['csrftoken']
+            data = {'username': args.username, 'password': args.password, 'csrfmiddlewaretoken': csrftoken}
+            headers = {'Cookie': 'csrftoken={0}'.format(csrftoken), 'Referer': login_url}
+
+            print("Logging into Platform to get the session id")
+            r2 = requests.post(login_url, verify=False, data=data, headers=headers, timeout=60)
```
[the remark this answers is not in the document]
Comment (mishalp-crest) [reply berryio_connector.py]: done

File: berryio_connector.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (mishalp-crest) [reply berryio_connector.py]: done

**Pull request merged**